docs/cloud/networks.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_postion: 2
+--- 
+
+
+## Managing Networks on Drangonfly Cloud
+
+Dragonfly Cloud allows you to create isolated networks where you can deploy your data stores similar to VPC's if you are familiar with AWS. 
```

Comment:
the motivation for a private network is security, latency and cost (as you avoid egress charges) 

it should proceed to connecting to the private network on the  "peering connection" page